client/ayon_substancepainter/plugins/publish/collect_textureset_images.py

```diff
@@ -187,10 +187,11 @@ def get_export_config(self, instance):
 
         creator_attrs = instance.data["creator_attributes"]
         preset_url = creator_attrs["exportPresetUrl"]
+        #TODO: temp fix only, find the better way for getting anatomy data
         instance.data["anatomyData"] = instance.context.data["anatomyData"]
-        self.log.debug(f"Exporting using preset: {preset_url}")
-
+        instance.data["taskEntity"] = instance.context.data["taskEntity"]
```

Comment:
This is not the correct fix - the context data is not necessarily the instance anatomy data.

Reply:
we need to figure out how to integrate with custom staging then.....